test/shjs.js

```diff
@@ -4,10 +4,9 @@ var assert = require('assert');
 
 function runScript(name) {
   // prefix with 'node ' for Windows, don't prefix for OSX/Linux
-  return shell.exec((process.platform === 'win32' ? 'node ' : '') +
-                    path.resolve(__dirname, '../bin/shjs') +
-                    ' ' +
-                    path.resolve(__dirname, 'resources', 'shjs', name), { silent: true });
+  var cmd = (process.platform === 'win32' ? 'node ' : '') + path.resolve(__dirname, '../bin/shjs');
```

Comment:
I'm not sure if I have already mentioned this, but the best way to spawn another `node` process is through `process.argv[0]`. On both Unix and Windows, `node` refers to the executable in PATH, while the user might have executed the script with the absolute path to another `node` executable. `process.argv[0]` guarantees that the current used Node.js is always used.